### PR TITLE
Market v10

### PR DESCRIPTION
**Pull request by tyshko5**
**Summary of changes**
Changes introduced in this pull request:
- 

**Reference issue to close (if applicable)**

Closes 

**Other information and links**

### CODE REVIEW

**Review by creativcoder — APPROVED** [review]

File: market_v10/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+fvm_ipld_hamt         = { workspace = true }
+fvm_shared            = { workspace = true, default-features = false }
+integer-encoding      = { workspace = true, default-features = false }
+libipld-core          = { workspace = true, features = ["serde-codec"] }
+log                   = { workspace = true }
+num-derive            = { workspace = true }
+num-traits            = { workspace = true }
+serde                 = { workspace = true, features = ["derive"] }
+fvm_ipld_amt = { workspace = true, features = ["go-interop"] }
+frc42_dispatch = "3.0.0"
```
Comment (creativcoder) [inline market_v10/Cargo.toml:27]: workspace = true

File: market_v10/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+frc46_token = "3.1.0"
```
Comment (creativcoder) [inline market_v10/Cargo.toml:28]: workspace = true

File: market_v10/Cargo.toml (hunk 1 above)
Comment (tyshko5) [reply market_v10/Cargo.toml]: Fixed

File: market_v10/Cargo.toml (hunk 2 above)
Comment (tyshko5) [reply market_v10/Cargo.toml]: Fixed

**Pull request merged**